src/go/vtunnel/test/e2e/connectivity_test.go

```diff
@@ -41,8 +41,8 @@ import (
 )
 
 var (
-	wslTarballName = "distro-0.21.tar"
-	wslTarballURL  = "https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/releases/download/v0.21/distro-0.21.tar"
+	wslTarballName = "distro-0.27.tar"
+	wslTarballURL = fmt.Sprintf("https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/releases/download/v0.27/%s", wslDistroName)
```

Comment:
Are referring to the variable that's assigned on the next line? Did you mean `wslTarballName` instead of `wslDistroName`?